src/extension.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path')
+
+const {workspace} = require('vscode')
+const {LanguageClient} = require('vscode-languageclient/node')
+
+/**
+ * @param {import('vscode').ExtensionContext} context
+ */
+function activate(context) {
+  const command = 'node'
+  const args = [path.join(__dirname, 'remark-language-server.js'), '--stdio']
+
+  /**
+   * @type {import('vscode-languageclient/node').ServerOptions}
+   */
+  const serverOptions = {
+    run: {command, args},
+    debug: {command, args: ['--inspect=6009', ...args]}
+  }
+
+  /**
+   * @type {import('vscode-languageclient').LanguageClientOptions}
+   */
+  const clientOptions = {
+    documentSelector: [{scheme: 'file', language: 'markdown'}],
+    synchronize: {
+      fileEvents: [
+        workspace.createFileSystemWatcher(
+          '**/.remark{ignore,rc,rc.cjs,rc.js,rc.json,rc.mjs,rc.yaml,rc.yml}'
+        ),
+        workspace.createFileSystemWatcher('**/package.json')
+      ]
+    }
+  }
+
+  const client = new LanguageClient(
+    'remarkLanguageServer',
+    'Remark Language Server',
```

Comment:
lowercase?

```suggestion
    'remark Language Server',
```
```suggestion
    'Remark Language Server',
```

Reply:
I think uppercase is better.

For reference, this is what other extensions use:

![image](https://user-images.githubusercontent.com/779047/147501010-96eae2e6-e9ba-4050-a4e4-5307c17cb669.png)

Reply:
But iPhone would be cased like that right? 🤷‍♂️
I'm 70% strongly in favor of always lowercase, but there's some wiggle room if you feel strongly about the other way

Reply:
It’s a human readable name for the language server. I don’t know why you feel strongly about using lower case here, but I feel pretty strong about consistency, which is to keep the current casing.

Reply:
Consistency is defined by the brand. `GitHub`, `GraphQL , `EditorConfig`, `npm`, `iPhone`, `macOS`, `remark`/`unified` etc are all, always, cased that way.
The Microsoft / VS Code npm package is also lowercase: https://marketplace.visualstudio.com/items?itemName=eg2.vscode-npm-script

Reply:
Oh I didn’t know _remark_ should be lower case! I’ll change it then. I’ll also change other places where _Remark_ is used.